src/osd/OSD.cc

```diff
@@ -3517,9 +3521,13 @@ int OSD::run_osd_bench_test(
       waiter.wait();
     }
   }
+  dout(0) << __func__
+          << " prefill took " << ceph_clock_now() - start
```

Comment:
Log with precision of 3 maybe?

Reply:
My plan is to leave it as visible as possible. Some tests run with debug_osd set to 0 and hence this info would be invisible with level = 3. 
Given this is a seldom op I don't think it's a big deal to have it at level 0. Or am I missing anything?

Reply:
@ifed01 I was not referring to the logging level. I am suggesting to use std::precision to truncate the decimal places for the duration to 3 places.